app/src/main/java/org/oppia/app/home/HomeActivityPresenter.kt

```diff
@@ -9,6 +9,8 @@ import org.oppia.app.activity.ActivityScope
 import org.oppia.app.drawer.NavigationDrawerFragment
 import javax.inject.Inject
 
+const val TAG_HOME_FRAGMENT = "TAG_HOME_FRAGMENT"
```

Comment:
It should like this:
`const val TAG_HOME_FRAGMENT = "HOME_FRAGMENT"`

Reply:
updated.